tools/dev/tests/data/diff.md

```diff
@@ -0,0 +1,8 @@
+- `provider1.resource2` removed
```

Comment:
what would be a nice addition here is actually doing a direct link to the reference api docs for each of the filters/actions/resources added

Reply:
Good point. I'll take a look.

Reply:
Added links.  Here's some sample output from the same invocation as above:

```markdown
# schema diff
- [`aws.elasticsearch-reserved`](https://cloudcustodian.io/docs/aws/resources/elasticsearch-reserved.html) added
- added common filters: [`reduce`](https://cloudcustodian.io/docs/aws/resources/aws-common-filters.html#aws-common-filters-reduce)
- `aws.elasticache-group`
  - added actions: [`delete`](https://cloudcustodian.io/docs/aws/resources/elasticache-group.html#aws-elasticache-group-actions-delete)
  - added filters: [`kms-key`](https://cloudcustodian.io/docs/aws/resources/elasticache-group.html#aws-elasticache-group-filters-kms-key)
- `aws.firehose`
  - added filters: [`kms-key`](https://cloudcustodian.io/docs/aws/resources/firehose.html#aws-firehose-filters-kms-key)
[...]
```

Let me know if any other ideas.

Reply:
I like it!  The only thing I might add is the supported resource count as it's a super impressive number but not required.  Seems to encompass the key features and will be very handy.  Thanks!